src/ui/components/App.tsx

```diff
@@ -89,7 +89,7 @@ function App({ theme, modal, children }: AppProps) {
   }, [theme]);
 
   useEffect(() => {
-    if (auth.isAuthenticated) {
+    if (auth.isAuthenticated && !isTest()) {
```

Comment:
why is this needed?

Reply:
Yeah, gonna be honest, I don't have a 100% understanding of this one. Something was interacting with updating or getting preferences, and that was failing in tests. I'm not sure if there's something special about how we auth in test such that auth.isAuthenticated returns true but user is not completely present or something? IDK!

Reply:
haha - fair enough, i could see why updating user settings as a test would be a no bueno...